F1/F1/Results/Repository/RacingResults.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  ResultsModel.swift
+//  F1
+//
+//  Created by Kaitlyn Sookdhew on 2024/04/02.
+//
+
+import Foundation
+
+// MARK: - RacingResults
+struct RacingResults: Codable {
+    let results: ResultsResponse
+
+    enum CodingKeys: String, CodingKey {
+        case results = "MRData"
+    }
+}
+
+// MARK: - ResultsResponse
+struct ResultsResponse: Codable {
+    let series, url, limit, offset, total: String
+    let raceTable: RaceTable
+
+    enum CodingKeys: String, CodingKey {
+        case series, url, limit, offset, total
+        case raceTable = "RaceTable"
+    }
+}
+
+// MARK: - RaceTable
+struct RaceTable: Codable {
+    let season: String
+    let races: [Race]
+
+    enum CodingKeys: String, CodingKey {
+        case season
+        case races = "Races"
+    }
+}
+
+// MARK: - Race
+struct Race: Codable {
+    let season, round, url, raceName: String
+    let circuit: Circuit
+    let date, time: String
+    let results: [RaceResult]
+
+    enum CodingKeys: String, CodingKey {
+        case season, round, url, raceName
+        case circuit = "Circuit"
+        case date, time
+        case results = "Results"
+    }
+}
+
+// MARK: - RaceResult
+struct RaceResult: Codable {
+    let number, position, positionText, points: String
```

Comment:
indenting. And please check that all your models follow the same convention. All strings on one line, or all separately. Same for all other types. You cleaned it up in some areas, but in areas like here, it follows a different convention